plugins/org.jkiss.dbeaver.ui.editors.image/src/org/jkiss/dbeaver/ui/data/managers/image/ImagePanelEditor.java

```diff
@@ -104,4 +112,10 @@ public void disposeEditor() {
 
     }
 
+    @Override
+    public File getExternalFile() {
+        return imageViewer.getExternalFile();
+    }
```

Comment:
Use the `control` argument instead of storing a reference to the viewer. Editors are singletons. You can't do that.